packages/@romejs/diagnostics/descriptions.ts

```diff
@@ -312,6 +312,10 @@ export const descriptions = createMessages({
         },
       ],
     }),
+    PREFER_BLOCK_STATEMENT: {
+      category: 'lint/preferBlockStatements',
+      message: 'TODO',
```

Comment:
Something simple like

```suggestion
      message: 'Block statements are preferred in this position',
```

Would probably do for now. A lot of the lint messages are pretty bad and shiuld be cleaned up prior to a release. It's easy for us to go over all the diagnostic messages now with this file.